docs/topology/docker-compose.yml

```diff
@@ -0,0 +1,175 @@
+version: '2'
```

Comment:
Rather than making a docs/topology maybe we should put this in the examples folder?

Reply:
I'll move the existing ports rewrite in examples to the test topologies one since this one is a better example for users.